test/integration/progressUsers.test.js

```diff
@@ -0,0 +1,187 @@
+const chai = require("chai");
+const sinon = require("sinon");
+
+const firestore = require("../../utils/firestore");
+const app = require("../../server");
+const authService = require("../../services/authService");
+
+const addUser = require("../utils/addUser");
+const cleanDb = require("../utils/cleanDb");
+const {
+  standupProgressDay1,
+  standupProgressDay2,
+  incompleteProgress,
+  stubbedModelProgressData,
+} = require("../fixtures/progress/userProgress");
+
+const userData = require("../fixtures/user/user")();
+
+const cookieName = config.get("userToken.cookieName");
+const { expect } = chai;
+
+// eslint-disable-next-line mocha/no-skipped-tests
+describe.skip("Test Progress Updates API", function () {
+  afterEach(async function () {
+    await cleanDb();
+  });
+
+  describe("Test the POST Call", function () {
+    let clock;
+    let testUserId;
+    let testUserJwt;
+    beforeEach(async function () {
+      clock = sinon.useFakeTimers({
+        now: new Date(2023, 5, 2).getTime(),
+        toFake: ["Date"],
+      });
+      testUserId = await addUser(userData[1]);
+      testUserJwt = authService.generateAuthToken({ userId: testUserId });
+    });
+
+    afterEach(function () {
+      clock.restore();
+    });
+
+    it("Stores the initial progress entry for the day and disregards any subsequent entries made before 6 am of the next day", async function () {
+      const responseFirstDay = await chai
```

Comment:
cant we directly add the data to db? rather than calling the API twice?

Reply:
I added an additional test that involves adding data from a database, pushing the data, and then checking it. Although this test may not be strictly necessary given the other tests, I decided to keep it as I believe it adds an extra layer of robustness to the testing.